packages/authgear-react-native/index.d.ts

```diff
@@ -30,7 +30,13 @@ export declare class CancelError extends AuthgearError {}
  * @public
  */
 export declare enum ColorScheme {
+  /**
+   * Use the light color scheme.

```

Comment:
Force to use the light color scheme in the AuthUI when the project config is "Auto".
Force to use the dark color scheme in the AuthUI when the project config is "Auto".

Reply:
okok